src/rooms/rooms.controller.ts

```diff
@@ -0,0 +1,28 @@
+import { Body, Controller, Get, Param, Post } from '@nestjs/common';
+import { RoomsService } from './rooms.service.js';
+
+@Controller()
+export class RoomsController {
+  constructor(private readonly rooms: RoomsService) {}
+
+  @Post('rooms')
+  create(@Body() body: { targetUrl: string }) {
+    const roomRecord = this.rooms.createRoom(body.targetUrl);
+    return {
+      roomId: roomRecord.roomId,
+      shareToken: roomRecord.token,
+      shareUrl: `http://localhost:3000/s/${roomRecord.token}`,
+      targetUrl: roomRecord.targetUrl,
+    };
+  }
+
+  @Get('share/:token')
+  getShare(@Param('token') token: string) {
+    const roomRecord = this.rooms.getRoomByToken(token);
+    if (!roomRecord) {
+      return { ok: false, error: 'Invalid token' };
+    } else {
+      return { ok: true, roomId: roomRecord.roomId, targetUrl: roomRecord.targetUrl };
+    }
```

Comment:
P3;
코드 가독성을 위해 if 조건문만 사용하는 코드 제안 드립니다

```suggestion
    if (!roomRecord) {
      return { ok: false, error: 'Invalid token' };
    } 
    
    return { ok: true, roomId: roomRecord.roomId, targetUrl: roomRecord.targetUrl };
```

Reply:
동의합니다. early return으로 정리해서 가독성 개선하겠습니다.